api/v2010/account/read-default/output/read-default.json

```diff
@@ -37,8 +37,8 @@
     }
   ],
   "uri": "/2010-04-01/Accounts.json?PageSize=50&Page=0",
-  "page_size": 50,
-  "start": 0,
-  "next_page_uri": null,
-  "page": 0
+  "page_size": "50",
+  "start": "0",
+  "next_page_uri": "",
+  "page": "0"
```

Comment:
Looking at the output of the actual API, the numbers shouldn't be strings and the `next_page_uri` and `previous_page_uri` should be `null` if they don't have a uri value.

```
> curl https://api.twilio.com/2010-04-01/Accounts.json -u "${TWILIO_ACCOUNT_SID}:${TWILIO_AUTH_TOKEN}"

{
  "first_page_uri": "/2010-04-01/Accounts.json?PageSize=50&Page=0",
  "end": 1,
  "previous_page_uri": null,
  "uri": "/2010-04-01/Accounts.json?PageSize=50&Page=0",
  "page_size": 50,
  "start": 0,
  "accounts": [...],
  "next_page_uri": null,
  "page": 0
}
```